# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1310854851 25200
#      Sat Jul 16 15:20:51 2011 -0700
# Node ID e51ad94535b9654d0fc2e529b2df92ceaf5d4321
# Parent  3035c2e130de91aac73ead53b04416458e845775
codesprint: Remove news.m from test statistics.

* news.m: Add %!assert(1) to remove from list of untested functions.

diff --git a/scripts/miscellaneous/news.m b/scripts/miscellaneous/news.m
--- a/scripts/miscellaneous/news.m
+++ b/scripts/miscellaneous/news.m
@@ -31,8 +31,12 @@ function news ()
     while (ischar (line = fgets (f)))
       puts (line);
     endwhile
   else
     error ("news: unable to locate NEWS file");
   endif
 
 endfunction
+
+
+## Remove from test statistics.  No real tests possible
+%!assert (1)
